scripts/openapi/collect_portal_frontend_urls.py

```
@@ -201,13 +314,13 @@ def render_markdown(urls: Sequence[FrontendUrl], language: str) -> str:
     intro = (
         "本文档由 `scripts/openapi/collect_portal_frontend_urls.py` 生成，用于跟踪 "
-        "Portal 前端 service 到 OpenAPI 的迁移进度。迁移完成后应删除。"
+        "Portal 前端 API 调用到 OpenAPI 的迁移进度。迁移完成后应删除。"
     )
     summary_title = "## 汇总"
-    service_title = "## 按 Service 汇总"
+    service_title = "## 按来源汇总"
     inventory_title = "## URL 清单"
     summary = [
-        f"- Service 文件数：{service_count}",
+        f"- 前端文件数：{service_count}",
```

Ruff (0.15.13)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)